rasa/cli/test.py

```diff
@@ -115,29 +116,51 @@ def run_core_test(args: argparse.Namespace) -> None:
     )
 
 
-def run_nlu_test(args: argparse.Namespace) -> None:
-    """Run NLU tests."""
+async def run_nlu_test_async(
+    config: Optional[Union[Text, Dict]],
+    data_path: Optional[Text],
+    models_path: Optional[Text],
+    output_dir: Optional[Text],
+    cross_validation: Optional[bool],
+    percentages: Optional[List[int]],
+    runs: Optional[int],
```

Comment:
As the our `argparse` config specifies a default, `None` can't happen

```suggestion
    runs: int,
```